docs/fiddles/features/web-usb/renderer.js

```diff
@@ -0,0 +1,29 @@
+async function testIt() {
+  const noDevicesFoundMsg = 'No devices found'
+  const grantedDevices = await navigator.usb.getDevices()
+  let grantedDeviceList = ''
+  if (grantedDevices.length > 0) {    
+    grantedDevices.forEach(device => {
+      grantedDeviceList += `<hr>${device.productName}</hr>`
```

Comment:
Here and on line 19, `productName` is optional so we should safeguard in the fiddles